projects/unbound/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER jsha@letsencrypt.org
+RUN apt-get update
+RUN apt-get install -y make libtool libssl-dev libexpat-dev
+RUN git clone --depth=1 https://github.com/NLnetLabs/unbound unbound
+WORKDIR unbound
+COPY fuzztest.c .
```

Comment:
fuzztest.c and fuzztest_seed_corpus.zip are in projects/unbound/ (or will be once I add the corpus.zip) and are getting copied into the root of the Unbound source directory inside the container. Happy to switch this up if there's a more standard way to do it.

Reply:
Oh, and there's definitely a weird thing about the corpus: It gets copied from projects/unbound/ into the image; then when running `build_image`, the image copies it into $OUT. It seems like there should be a better way to get a file from projects/unbound/ into $OUT but I didn't immediately see one.

Also, I realized the corpus is actually committed, but GitHub hides it because it's binary.

Reply:
Sorry about this, was definitely confused here. Thanks for explaining.

Reply:
Thanks for explaining looks like i was confused here.

> > Can you make the file a `.cc` and use `clang++` to compile (where `region` is `static`).
> 
> I'd prefer the approach I've got here, mainly because the upstream is all C, no C++, so doing this would make it harder to upstream.

OK then what do you think of something like this:
```
struct regional * region = NULL;

int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len)
{
  if (!region) {
    region = regional_create();
    if (!region) abort();
}
...
}
```
Or you can use a variable like `bool initialized`.